src/common/Color.cpp

```diff
@@ -141,14 +122,8 @@ std::string StripColors( const std::string& input )
 
 	for ( const auto& token : Parser( input.c_str() ) )
 	{
-		if ( token.Type() == Token::TokenType::CHARACTER )
-		{
-			output.append( token.Begin(), token.Size() );
-		}
-		else if ( token.Type() == Token::TokenType::ESCAPE )
-		{
-			output.push_back(Constants::ESCAPE);
-		}
+		Str::StringView text = token.PlainText();
```

Comment:
Given that we are touching this code, can this be an opportunity to do:

```
std::string StripColors( Str::StringRef in ) {
  std::string output;
  output.resize(in.size()); // output basically guaranteed to be <= input since we will never add any chars
  StripColors(in.c_str(), output.c_str(), output.size());
  return output;
}

Reply:
You also would need a `strlen` there to reduce the output string to its actual size so I think that would be longer/uglier than what we have now.

I added a commit to pre-allocate the string buffer if that's what you're going for.